cf-serverd/cf-serverd-enterprise-stubs.c

```diff
@@ -58,6 +58,11 @@ ENTERPRISE_FUNC_1ARG_DEFINE_STUB(bool, ReceiveCollectCall, ARG_UNUSED ServerConn
     return false;
 }
 
+ENTERPRISE_FUNC_1ARG_DEFINE_STUB(bool, ReturnCookies, ARG_UNUSED ServerConnectionState *, conn)
```

Comment:
*"This command is used for hosts using duplicate identity."* is a bit misleading, I think. After all, the protocol is used for all hosts, if possible, and it's **purpose** is to identify hosts with duplicate identity. Also the changelog entry should be more explanatory, I think.

Reply:
> ~"This command is used for hosts using duplicate identity."~ is a bit misleading

~English~ is a bit misleading.

Natural language is a bit misleading.

> "This command is used for **detecting** hosts using duplicate identity."

I see that I made a mistake, it is missing `detecting`.

> After all, the protocol is used for all hosts, if possible and it's **purpose** is to identify hosts with duplicate identity. Also the changelog entry should be more explanatory, I think.

Again, see above. I think, after adding `detecting`, it makes sense. Added.